src/coreclr/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -456,7 +456,7 @@ unsafe partial class CorInfoImpl
     {
         private const CORINFO_RUNTIME_ABI TargetABI = CORINFO_RUNTIME_ABI.CORINFO_CORECLR_ABI;
 
-        private uint OffsetOfDelegateFirstTarget => (uint)(3 * PointerSize); // Delegate::m_functionPointer
+        private uint OffsetOfDelegateFirstTarget => (uint)(2 * PointerSize); // Delegate::m_functionPointer
```

Comment:
This is R2R breaking change. It would min supported R2R version bump.

Reply:
Yeah I've suspected so, do we want to wait with this then?

Reply:
You may be able to avoid it by swapping the order of `_methodPtr` and `_methodPtrAux`

Reply:
Do we want to swap them back on the next bump (since AFAIK having the fields consecutive in memory might be slightly better for perf)?

Reply:
Sounds reasonable to me.